code-challenges.js

```diff
@@ -57,12 +151,41 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+console.log(myNumbers.lastIndexOf(42))
```

Comment:
Keep in mind you can also pass the variables into lastIndexOf instead of the number.